mentoring/README.md

```diff
@@ -1,13 +1,8 @@
 ---
-Title: Kubernetes Upstream Mentoring Programs
----
-
-# Kubernetes Upstream Mentoring Programs
```

Comment:
It's kind of annoying, when viewing on the contributor site, but we should still leave this for folks who find it here in GitHub =/